etcd/cmd/microshift-etcd/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/component-base/cli"
+	"k8s.io/component-base/logs"
+)
+
+func main() {
+	logs.InitLogs()
+	defer logs.FlushLogs()
+
+	cmd := &cobra.Command{
+		Use:   "microshift-etcd",
+		Short: "MicroShift, a minimal OpenShift",
```

Comment:
nit: We probably want a different description string?

Reply:
it's not user facing app so I removed it altogether